unpackaged/post/reports/reports/ProgramManagementReportsUnpackaged/Unique_Contacts_Receiving_Service_Year_t_44q.report

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Report xmlns="http://soap.sforce.com/2006/04/metadata">
+    <columns>
+        <aggregateTypes>Unique</aggregateTypes>
+        <field>%%%NAMESPACE%%%ServiceDelivery__c.%%%NAMESPACE%%%Contact__c</field>
+    </columns>
+    <columns>
+        <field>CUST_NAME</field>
+    </columns>
+    <columns>
+        <field>%%%NAMESPACE%%%ServiceDelivery__c.%%%NAMESPACE%%%DeliveryDate__c</field>
+    </columns>
+    <columns>
+        <aggregateTypes>Sum</aggregateTypes>
+        <field>%%%NAMESPACE%%%ServiceDelivery__c.%%%NAMESPACE%%%Quantity__c</field>
+    </columns>
+    <columns>
+        <field>%%%NAMESPACE%%%ServiceDelivery__c.%%%NAMESPACE%%%UnitOfMeasurement__c</field>
+    </columns>
+    <description>How many unique Contacts have received a Service?</description>
+    <format>Summary</format>
+    <groupingsDown>
+        <aggregateType>Unique</aggregateType>
+        <dateGranularity>Day</dateGranularity>
+        <field>%%%NAMESPACE%%%ServiceDelivery__c.%%%NAMESPACE%%%Service__c</field>
+        <sortByName>%%%NAMESPACE%%%ServiceDelivery__c.%%%NAMESPACE%%%Contact__c</sortByName>
+        <sortOrder>Asc</sortOrder>
+        <sortType>Aggregate</sortType>
+    </groupingsDown>
+    <name>Unique Contacts Receiving Service</name>
```

Comment:
Do we need to filter this report to show only the service deliveries for the current year? There is an AC in the user story for this. If not we can remove that from the AC so it is easier for QE. Also one more thing I noticed is that the rowCount is displayed. I think it could be confusing for the users since the rowCount will be the number of service delivery records created for that unique contact but here the report title just talks about the unique contacts receiving the service.

Reply:
Nope, we discussed this on Slack. I'll update the WI.

Reply:
Shontese liked the row count.